catalog/nsili/catalog-nsili-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -76,6 +79,9 @@
             name="Poll Interval" id="pollInterval"
             required="true" type="Integer" default="5"/>
 
+        <AD description="Maximum parallel threads for converting results and retrieving thumbnails" name="Max Hit Count"
+            id="numberWorkerThreads" required="true" type="Integer" default="4"/>
```

Comment:
I see that these are added to metatype, but are they linked to blueprint and actually used?